modules/040-node-manager/templates/early-oom/daemonset.yaml

```diff
@@ -50,6 +50,13 @@ spec:
         app: early-oom
       name: early-oom
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node.deckhouse.io/memory-reserved
+                operator: DoesNotExist
```

Comment:
In the discussion we determined that the user should turn off early-oom, so we do not need to do this.

Reply:
Fixed.